Client/Frontend/Settings/SettingsViewController.swift

```diff
@@ -333,7 +333,7 @@ class SettingsViewController: TableViewController {
     
     if UIDevice.current.userInterfaceIdiom == .pad {
       tabs.rows.append(
-        Row(text: Strings.showTabsBar, image: UIImage(named: "settings-show-tab-bar", in: .module, compatibleWith: nil)!.template, accessory: .switchToggle(value: Preferences.General.tabBarVisibility.value == TabBarVisibility.always.rawValue, { Preferences.General.tabBarVisibility.value = $0 ? TabBarVisibility.always.rawValue : TabBarVisibility.never.rawValue }), cellClass: MultilineValue1Cell.self)
+        Row(text: Strings.showTabsBar, image: UIImage(named: "settings-show-tab-bar", in: .module, compatibleWith: nil)!.template, accessory: .switchToggle(value: Preferences.General.tabBarVisibility.value != TabBarVisibility.never.rawValue, { Preferences.General.tabBarVisibility.value = $0 ? TabBarVisibility.always.rawValue : TabBarVisibility.never.rawValue }), cellClass: MultilineValue1Cell.self)
```

Comment:
maybe break this long line, hard to notice the toggle logic at first glance

Reply:
Haha easier to read now, but harder to tell what changed since git cant highlight the changed part in the line :P